hybrasyl/Objects/User.cs

```diff
@@ -262,7 +263,8 @@ public bool IsPrivileged
             {
                 if (Game.Config.Access?.Privileged != null)
                 {
-                    return IsExempt || Flags.ContainsKey("gamemaster") || Game.Config.Access.Privileged.Contains(Name);
+                    return IsExempt || Flags.ContainsKey("gamemaster") ||

```

Comment:
not sure what this has to do with throttles

Reply:
Nothing at all, but it is something that needed to be cleaned up for testing, so it got cleaned up. Basically makes the value of `<Privileged>` in XML case insensitive.

Reply:
+1